client/gatsby-browser.js

```diff
@@ -27,3 +28,9 @@ wrapRootElement.propTypes = {
 export const wrapPageElement = layoutSelector;
 
 export const disableCorePrefetching = () => true;
+
+export const onClientEntry = () => {
+  // purge the _csrf cookie, rather than relying what the browser decides a
+  // Session duration is
+  cookies.erase('_csrf');
+};
```

Comment:
@raisedadead is this what you had in mind?  For reference: [onClientEntry](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#onClientEntry)

Reply:
Seems reasonable. Will try this out on staging later today.

Reply:
I might be missing something, but this does not seem to delete the `_csrf`  locally, possibly because onClientEntry runs before the cookie is set?